copilot/fsd-authenticator/manifest.yml

```diff
@@ -70,20 +70,40 @@ secrets:
   AZURE_AD_TENANT_ID: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/AZURE_AD_TENANT_ID
 
 # You can override any of the values defined above by environment.
-environments:
+environments:  test:
+  deployment:
+    rolling: 'recreate'
   uat:
     variables:
       ALLOW_ASSESSMENT_LOGIN_VIA_MAGIC_LINK: true
```

Comment:
This was here already, but why is it here for UAT only?  (Probably just I don't understand what it's for)

Reply:
The magic link part?

We did go through this as part of this build - in cloudfoundry, only UAT has this environment variable set

cf env funding-service-design-authenticator-dev
Not set

cf target -s sandbox
cf env funding-service-design-authenticator-test
Not set

cf target -s uat
cf env funding-service-design-authenticator
ACCOUNT_STORE_API_HOST: xxx
ALLOW_ASSESSMENT_LOGIN_VIA_MAGIC_LINK: true 

cf target -s production
cf env funding-service-design-authenticator
ACCOUNT_STORE_API_HOST: xxx
ALLOW_ASSESSMENT_LOGIN_VIA_MAGIC_LINK: false

Actually, you make a good point - the _default_ is set to true, so this should probably be set to true for both dev and test 🤔

Reply:
https://github.com/communitiesuk/funding-service-design-authenticator/blob/main/config/envs/default.py#L106C5-L108 (default set to true)

Reply:
Added in dev and test